job-service-db/src/main/resources/v2.3.0/changelog.xml

```diff
@@ -45,7 +45,7 @@
                 <constraints nullable="false"/>
             </column>
             <column name="target_pipe" type="VARCHAR(255)">
-                <constraints nullable="false"/>
+                <constraints nullable="true"/>
```

Comment:
You can't change a change set for a version that has already been released.  You have to handle this in the v2.6.0 changelog.